cmd/schema.go

```diff
@@ -289,21 +289,19 @@ func getSchemaCmd() *cobra.Command {
 
 			fmt.Printf("Schema successfully written to %s\n", output)
 
-			if metadata {
-				if resMetadata != nil {
-					report := [][]string{}
+			if metadata && resMetadata != nil {
```

Comment:
@ishanarya0 More simpler approach for this could also be to return if `resMetamdata` is `nil` and `metadata` is not there. so that success flow can come without an if condition.

Reply:
Yaa, that would look better. Done!